.github/workflows/build_and_test.yml

```diff
@@ -499,7 +499,7 @@ jobs:
           - >-
             pyspark-sql, pyspark-resource, pyspark-testing
           - >-
-            pyspark-core, pyspark-errors, pyspark-streaming
+            pyspark-core, pyspark-errors, pyspark-streaming, pyspark_logging
```

Comment:
Will this affect branch-3.5 daily test?

Reply:
IIRC, it won't affect 3.5.
Let's see, I will monitor the 3.5 daily and fix if it fails.

Reply:
OK

Reply:
confirmed in https://github.com/apache/spark/actions/runs/12643188982/job/35228825941,
it seems pyspark test commands automatically skip non-existent modules.